pages/accountInformation.js

```diff
@@ -0,0 +1,45 @@
+const {faker} = require("@faker-js/faker");
+const COMMON_TIMEOUT = global.__TIMEOUT__;
+
+
+const SELECTORS = {
+    FIRST_NAME_INPUT: "//input[@id='firstname']",
+    LAST_NAME_INPUT: "//input[@id='lastname']",
+    SAVE_BUTTON: "//button[@class='action save primary']",
+};
+
+const firstNameInput = async (page, firstName = faker.name.firstName('male')) => {
```

Comment:
What this function does? The name of the function should describe it. I assume that it sets name so change it to `setFirstName`